Sources/BraintreePayPal/BTPayPalRequest.m

```diff
@@ -53,6 +53,10 @@ - (NSString *)landingPageTypeAsString {
         parameters[@"merchant_account_id"] = self.merchantAccountID;
     }
 
+    if (self.correlationId) {
```

Comment:
Should this be `clientMetadataId` instead? That's what a merchant will get from the GraphQL call.

Reply:
I think correlation id is much clearer and aligns with what the value actually is: an ID that correlates a bunch of data (in this case, STC data and RDA data). I am not sure why graphQL chose `clientMetadataId` :(

Even the value that the gateway takes is called `correlation_id`